src/components/Tasks/Toolbar.tsx

```diff
@@ -20,23 +24,66 @@ import { StoreState } from '../../store';
 
 import Menu from '../../widgets/Menu';
 
+const transitionTimeout = 500;
+
+const transitionStyles = {
+  entering: { visibility: 'visible', width: '100%', overflow: 'hidden' },
+  entered: { visibility: 'visible', width: '100%' },
+  exiting: { visibility: 'visible', width: '0%', overflow: 'hidden' },
+  exited: { visibility: 'hidden', width: '0%' },
+};
+
+const useStyles = makeStyles((theme) => ({
+  button: {
+    marginRight: theme.spacing(1),
+  },
+  autocomplete: {
+    transition: `width ${transitionTimeout}ms`,
+    marginRight: theme.spacing(1),
+  },
+  input: {
+    borderRadius: theme.shape.borderRadius,
+
+    '&::before, &:hover::before, &::after': {
+      borderBottom: 0,
+    },
+
+    '& input': {
+      padding: '0.75em',
+    },
+  },
+  adornment: {
+    marginTop: '0!important',
```

Comment:
`!important` is a code-smell. What do you need it for? Why do you need extra styling?

Reply:
No longer necessary, now that I've cleaned up those styles.